### PR TITLE
DLTS → DTLS

### LINKED ISSUE #20908: DLTS (not DTLS) ?
OpenSSL refers to **DLTS** in two places:

https://github.com/openssl/openssl/blob/3868807d2fe5a72aa897ce5f7f7ba7e9cc3c09cb/doc/man1/openssl.pod?plain=1#L590-L592

https://github.com/openssl/openssl/blob/3868807d2fe5a72aa897ce5f7f7ba7e9cc3c09cb/test/dtlstest.c#L621-L624

I am not a crypto specialist, and I wonder what DLTS is. A typo? Some real DTLS alternative? If so, where can I find information on DLTS?

### PR DESCRIPTION
**Pull request by DimitriPapadopoulos**
CLA: trivial

Fix a typo that is confusing for newcomers. It would be great if it could be merged into the 3.0 branch.

Fixes #20908.

##### Checklist

- [x] documentation is added or updated
- [x] tests are added or updated

### CODE REVIEW

File: doc/man1/openssl.pod [hunk 1]
```diff
@@ -589,7 +589,7 @@ B<s_client> and B<s_server> commands.
 
 =item B<-dtls>, B<-dtls1>, B<-dtls1_2>
 
-These options specify to use DTLS instead of DLTS.
+These options specify to use DTLS instead of TLS.
```
Comment (DimitriPapadopoulos) [inline doc/man1/openssl.pod:592]: Reviewers: This part of the fix I am not 100% certain about.

**Review by hlandau — APPROVED** [review]

File: doc/man1/openssl.pod (hunk 1 above)
Comment (t8m) [reply doc/man1/openssl.pod:592]: The fix is correct.

**Review by t8m — APPROVED** [review]
OK with CLA: trivial

**Comment by t8m** [comment]
@hlandau Please confirm that you're ok with CLA: trivial.

**Comment by hlandau** [comment]
Agree trivial.

**Review by paulidale — APPROVED** [review]
fine with trivial

**Comment by hlandau** [comment]
Merged to master, 3.0 and 3.1. Thank you.

The test change was not relevant to 3.1 and 3.0 so not included during cherrypicking.

**Pull request closed without merging**